pkg/operator/ceph/csi/spec.go

```diff
@@ -97,6 +101,16 @@ func CSIEnabled() bool {
 	return EnableRBD || EnableCephFS
 }
 
+func UpdateCSIDriverNames(namespace string) {
+	prefix := CSIParam.DriverNamePrefix
+	if CSIParam.DriverNamePrefix == "" {
```

Comment:
Similar logic for this is in the StartCSIDrivers() method. Can you just set these from inside that method?

Reply:
Should this method be removed now?

Reply:
updated now